src/App.js

```diff
@@ -155,13 +155,15 @@ export default function App() {
                 {/* Hero unit */}
                 <div className={classes.heroContent}>
                     <Container maxWidth="md">
-                            <LanguageBar 
-                              inputLanguage={inputLanguage} 
-                              outputLanguage={outputLanguage} 
-                              handleInputLanguageChange= {handleInputLanguageChange} 
-                              handleOutputLanguageChange= {handleOutputLanguageChange} 
-                              classes={classes}
-                            />
+                            { Object.keys(supportedLanguages).length > 0 &&
```

Comment:
Nit: consider some loading state so the user knows what's going on.

Reply:
Already have [ticket](https://tree.taiga.io/project/jackdavidweber-cjs-capstone/us/136?no-milestone=1)!